roadmap/implementers-guide/src/node/disputes/dispute-coordinator.md

```diff
@@ -389,14 +387,26 @@ increment a counter for each signing participant of explicit `invalid` votes.
 What votes do we treat as a potential spam? A vote will increase a spam slot if
 and only if all of the following condidions are satisfied:
 * the candidate under dispute is not included on any chain
+* the candidate under dispute is not backed on any chain
 * the dispute is not confirmed
 * we haven't casted a vote for the dispute
 
+Whenever any vote on a dispute is imported these conditions are checked. If the
+dispute is found not to be potential spam, then spam slots for the disputed candidate hash are cleared. This decrements the spam count for every validator
+which had voted invalid. 
+
+To keep spam slots from filling up unnecessarily we want to clear spam slots 
+whenever a candidate is seen to be backed or included. Fortunately this behavior
+is acheived by clearing slots on vote import as described above. Because on chain
+backing votes are processed when a block backing the disputed candidate is discovered, spam slots are cleared for every backed candidate. Included 
+candidates have typically also been seen as backed, so decrementing spam slots is
```

Comment:
not typically - always. If we have seen a fork including a candidate, it must have been backed previously on that fork.

Another thing, while I am fine with the guide describing this - this is something that should be documented in the source code as well, so it is clear for any editor that we rely on those side effects.